src/randoop/types/TypeNames.java

```diff
@@ -0,0 +1,50 @@
+package randoop.types;
+
+import randoop.util.PrimitiveTypes;
+
+import plume.UtilMDE;
+
+public class TypeNames {
```

Comment:
Please add Javadoc.  Also, should this be called "ClassNames" since that is what it manipulates?